samza-core/src/main/scala/org/apache/samza/diagnostics/DiagnosticsManager.java

```diff
@@ -80,20 +82,20 @@
   private final SystemStream diagnosticSystemStream;
 
   public DiagnosticsManager(String jobName, String jobId, Map<String, ContainerModel> containerModels,
```

Comment:
For a constructor with this many args and a good chance for needing to add more, consider expanding to one line per argument. Then additions in the future will be easier to read, and it will be easier to keep the line lengths short.